### ISSUE TITLE
Proposed Business 58 — Personal Writing Voice Studio Phase 1 visual UI reference

### ISSUE THREAD

**Issue by skerishKang** [skerishKang/ai-revenue-lab#219]
## Purpose

Create the Phase 1 visual UI reference for proposed Business 58 — **Personal Writing Voice Studio / 나의 문체 스튜디오**.

Parent queue: #154
Product decision: #218
Related translation product: #216 / #217

This is a polished static visual reference using only repository-local synthetic writing and verified public-domain examples. It is not an operational upload service, model-training environment, accepted UX or backend.

## Product promise

> 사용자가 권리를 가진 자신의 글이나 정식 계약된 원고를 업로드하면, 그 사람의 문장 호흡·어휘 선택·서술 거리·대화 리듬·이미지 사용·구성 습관을 개인 문체 프로필로 분석하고, 그 프로필을 번역·새 글 작성·개작·편집에 적용하되 원문 권리와 생성 내역을 사용자가 통제하게 한다.

## Visual direction — Authored Voice Atelier / 문체 아틀리에

The interface should feel like a private author–editor studio and controlled literary laboratory, not an AI chat, generic file uploader, fine-tuning dashboard or public celebrity-style marketplace.

Use:

- editorial manuscript surfaces and restrained proofing marks;
- corpus folios, voice evidence strips, sentence-rhythm diagrams and margin annotations;
- warm neutral paper with deep ink and one restrained contract/status accent;
- clear separation between observed evidence, inferred trait and user-approved rule;
- visible rights and consent state at every relevant screen;
- Korean-first labels with compact English terminology where useful;
- system serif for long-form samples and system sans for controls;
- no external fonts, gradients, glassmorphism, chat bubbles, neon AI imagery or rounded-card overload.

## Required fixture strategy

Use exactly two demo profiles:

### A. Synthetic contracted writer

Create a clearly fictional contemporary Korean writer with a repository-local synthetic corpus. Label all texts as synthetic, consented and reference-only.

Suggested profile:

```text
Name: 윤해진
Role: fictional essayist and screenwriter
Corpus: 12 synthetic documents / 38,400 synthetic Korean characters
Rights state: demo contract active
Shared-model training: prohibited
Public attribution: prohibited
```

### B. Verified public-domain writer

Use one author whose source works are verified public domain in the intended territory. Do not copy a protected modern translation, commentary, edition notes or branding. Record source and rights evidence.

The public-domain profile demonstrates historical voice analysis only. It must not imply endorsement, authenticity of new works or an estate relationship.

## Required reference research

Document official or primary references for:

- Korean copyright term and derivative/translation rights;
- WIPO/Berne author and translation rights;
- current generative-AI training and copyright analysis from an official copyright authority;
- consent, withdrawal and data-governance patterns for creator AI tools;
- model fine-tuning/adapters versus retrieval/prompt profiles;
- authorship and disclosure for AI-assisted outputs;
- private author/publisher workflow and proofing references.

Review 3–5 adjacent products or workflows and explain why this differs from:

- generic custom GPT/fine-tuning consoles;
- Grammarly-style rewriting;
- public celebrity imitation tools;
- Business 57 Classic Literature Translation Studio;
- Personal Media Studio;
- AI Content Engine.

Record this in:

```text
reference/business-58-personal-writing-voice-studio-v1/REFERENCE_NOTES.md
```

### 1. Rights-first corpus intake / 권리 확인 코퍼스 접수

Show a static review state representing:

- document inventory;
- uploader role: author, rightsholder, authorized representative or public-domain researcher;
- allowed purposes: analysis, translation, generation, editing, evaluation;
- storage and retention term;
- shared-model training set to off;
- withdrawal/deletion condition;
- confidential-contract state.

Do not implement a real upload. The state is a visual contract preview using synthetic metadata.

### 2. Corpus evidence room / 문체 근거실

Show representative manuscript fragments with evidence markers for:

- sentence length and variation;
- paragraph rhythm;
- recurring words;
- narrator distance;
- dialogue density;
- image/metaphor use;
- punctuation habits;
- scene versus exposition balance.

Observed evidence and inferred interpretation must be visually distinct.

### 3. Personal voice profile / 개인 문체 프로필

Present an inspectable profile, not a personality horoscope.

Include:

- evidence-backed traits;
- confidence and coverage;
- counterexamples;
- user/editor corrections;
- approved, rejected and unresolved traits;
- profile version and corpus version.

At least one inferred trait must be rejected by the author in the fixture to demonstrate control.

### 4. Voice-applied translation / 내 문체로 번역

Show one short public-domain source passage as:

- neutral faithful Korean translation;
- authorized synthetic writer voice version;
- explanation of what changed in rhythm, diction and narrative distance;
- source fidelity warnings where the voice layer changes nuance.

This state demonstrates the bridge to Business 57 without merging the products.

### 5. Voice-applied generation / 내 문체로 새 글

Show:

- neutral brief;
- generated draft in the synthetic writer’s authorized voice;
- evidence links to corpus patterns;
- originality/provenance notice;
- author review status;
- explicit statement that the result is AI-assisted and not author-approved until approval is recorded.

### 6. Contract and deletion controls / 계약·철회·삭제

Show a contract lifecycle state with:

- active purposes;
- prohibited purposes;
- term and territory;
- public attribution permission;
- shared-model use off;
- export history;
- revoke access action;
- corpus deletion receipt;
- adapter/profile deletion receipt;
- outputs retained or withdrawn according to the fixture contract.

This is a static visual state; no destructive operation is implemented.

### 7. Signature motion — Voice Trace / 문체 흔적

Create one restrained `680ms` motion that traces three or four visible corpus features into the final voice-applied sentence.

Requirements:

- source manuscripts and final output geometry remain stable;
- only evidence threads, emphasis strokes and trait markers animate;
- no whole-screen fade or layout shift;
- focus and scroll remain stable;
- deterministic replay;
- reduced-motion mode presents the final complete evidence mapping immediately.

## Public-domain and synthetic safety

The UI must visibly state:

- synthetic contracted writer: fictional and consented for demo;
- public-domain author: source status verified, no endorsement or estate relationship;
- no living author’s unlicensed corpus is used;
- upload does not itself prove rights;
- shared-model training is off by default;
- private or licensed corpora must not enter the public repository.

## Required content examples

Include:

- at least six synthetic manuscript fragments across different dates or genres;
- at least one counterexample that weakens a proposed voice trait;
- one author correction to the inferred profile;
- one neutral translation and one voice-applied translation;
- one neutral brief and one voice-applied generated draft;
- one divergence warning where voice preservation conflicts with source fidelity;
- one consent scope change;
- one deletion/withdrawal receipt preview.

## Accessibility and responsive requirements

Validate at:

- desktop `1440×1100`;
- tablet `768×1024`;
- mobile `390×844`;
- reduced-motion desktop.

Required:

- horizontal overflow `0`;
- keyboard state navigation;
- visible focus;
- accessible names for contract, profile and comparison controls;
- no information conveyed only through colour;
- long-form samples remain readable;
- console/page errors `0`;
- failed local assets `0`;
- external runtime requests `0`.

## Required source and evidence files

```text
reference/business-58-personal-writing-voice-studio-v1/
├── index.html
├── README.md
├── REFERENCE_NOTES.md
├── RIGHTS_AND_CONSENT_SPEC.md
├── VOICE_PROFILE_SPEC.md
├── MOTION_SPEC.md
├── IMAGE_SOURCES.md
├── assets/**
├── styles/**
├── scripts/**
└── evidence/**
```

Committed evidence must include:

- static contract validator;
- browser validator;
- validation JSON;
- exact run commands;
- deterministic asset-version check;
- rights/source manifest validation;
- key desktop screenshots;
- actual mobile screenshot;
- Voice Trace motion evidence;
- reduced-motion final state.

## Allowed scope

```text
reference/business-58-personal-writing-voice-studio-v1/**
```

Do not modify:

- `apps/**`;
- another Business workspace;
- registry/backlog or repository-wide docs;
- CI or deployment configuration.

## Target branch

```text
feat/business-58-personal-writing-voice-studio-ui
```

## Target Draft PR title

```text
feat(personal-writing-voice): add Phase 1 visual UI reference
```

## Phase boundary

This issue is `UI_ONLY`.

Do not implement:

- actual uploads;
- private corpus storage;
- model training, fine-tuning or adapter generation;
- authentication or tenant isolation;
- public named living-author profiles;
- publishing, billing or commercial release;
- UX acceptance or backend APIs;
- Cloudflare deployment before exact-head `UI_APPROVED` and separate user authorization;
- Ready transition, merge or Issue closure without Web CTO review.

## Proposed deployment after approval

```text
Cloudflare Pages project:
ai-revenue-business-58-personal-writing-voice-studio

Production URL:
https://ai-revenue-business-58-personal-writing-voice-studio.pages.dev/
```

Public deployment, if later approved, must contain only synthetic and verified public-domain examples. Contracted living-author demonstrations remain private and outside the public repository/deployment.

## Acceptance criteria

- seven materially distinct states exist;
- rights-first intake and deletion lifecycle are visible;
- voice profile is evidence-based and correctable;
- translation and generation applications are both demonstrated;
- no unlicensed living-author corpus or named imitation preset appears;
- synthetic/public-domain disclosures are explicit;
- all required viewports and reduced motion pass;
- all source changes remain inside the allowed workspace;
- Draft PR remains open and unmerged;
- final worker status is `UI_REVIEW_READY`, not `UI_APPROVED`.

**Comment by skerishKang** [comment]
Implemented — capability verified in main. Triage verdict: IMPLEMENTED (evidence anchor in #2018 batch table; sample set CTO-verified: PR #1585 merged 2026-09-02, 7 module files present, 44 reference folders present). Closed per CTO execution of the #2018 triage.
